app/client/cypress/integration/home/links.spec.js

```diff
@@ -17,4 +17,28 @@ describe('Homepage', () => {
     cy.findByText('National').click();
     cy.url().should('include', '/national');
   });
+  it(`"Data" button links to the data page`, () => {
+    cy.findByText('Data').click();
+    cy.url().should('include', '/data');
+  });
+  it(`"About" button links to the about page`, () => {
+    cy.findByText('About').click();
+    cy.url().should('include', '/about');
+  });
+  it(`"Contact Us" button links to the contact us page in a new window`, () => {
+    // Cypress does not have multi-tab support, verify the href and target, don't click
+    cy.findByText('Contact Us')
+      .should('have.attr', 'href')
+      .and(
+        'equal',
+        'https://www.epa.gov/waterdata/forms/contact-us-about-hows-my-waterway',
+      );
+    cy.findByText('Contact Us')
+      .should('have.attr', 'target')
+      .and('equal', '_blank');
```

Comment:
You can simplify this by removing your use of `.and()` and passing the value as a third argument to `should()`. You can chain subsequent `should()` calls as they return the subject. nytime you're testing that a link opens in a new tab, you also want to test that it has the suggested "rel" attributes too (for security best practices). And finally, I'm updating the comment just to be consistent with the comments used in other tests that are testing similar functionality.

```suggestion
    // since Cypress doesn't support multiple tabs, we'll do the next best thing
    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
    cy.findByText('Contact Us')
      .should(
        'have.attr',
        'href',
        'https://www.epa.gov/waterdata/forms/contact-us-about-hows-my-waterway',
      )
      .should('have.attr', 'target', '_blank')
      .should('have.attr', 'rel', 'noopener noreferrer');
```